dfp.py:6: UserWarning: viewer requires Qt
  from skimage.viewer import ImageViewer
dfp.py:6: UserWarning: viewer requires matplotlib<3.5; please consider moving to another tool such as napari as the viewer module will be removed in version 1.0.
  from skimage.viewer import ImageViewer
Using TensorFlow backend.
Traceback (most recent call last):
  File "dfp.py", line 22, in <module>
    from vizdoom import DoomGame, ScreenResolution
  File "/dcs/22/u2256152/.conda/envs/test/lib/python3.7/site-packages/vizdoom/__init__.py", line 12, in <module>
    from .vizdoom import __version__ as __version__
ImportError: libicudata.so.67: cannot open shared object file: No such file or directory
